config/osu.php

```diff
@@ -93,6 +93,17 @@
         'notice' => presence(str_replace('\n', "\n", env('STORE_NOTICE'))),
     ],
     'twitch_client_id' => env('TWITCH_CLIENT_ID'),
+    'tournament_support' => [
```

Comment:
Hrm, maybe `tournament_banner(s)` for these variables makes more sense than `tournament_support`?